keras_cv/layers/preprocessing/invert.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+class Invert(tf.keras.layers.Layer):
+    """Inverts the image pixels.
+
+    Usage:
+        ```
+        invert = Invert()
+        (images, labels), _ = tf.keras.dataset.cifar10.load_data()
+        # Note that images are Tensor with values in the range [0, 255]
+        images = invert(images)
```

Comment:
we might want to add some numerical value here, eg [100, 150, 200] is invert to [155, 105, 55]

Reply:
Added printing of the three first values across channels.